WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -439,7 +458,16 @@ private extension OrderDetailsDataSource {
     private func configureShippingLabelProduct(cell: ProductDetailsTableViewCell, at indexPath: IndexPath) {
         cell.selectionStyle = .default
 
-        // TODO-2167: show aggregated order item (product) for a shipping label
+        guard let shippingLabel = shippingLabel(at: indexPath) else {
+            return
+        }
+
+        guard let orderItem = shippingLabelOrderItemsAggregator?.orderItems(of: shippingLabel)[safe: indexPath.row] else {
```

Comment:
ℹ️ We can also have a helper that encapsulates `safe` inside it. 

```swift
    func orderItem(of shippingLabel: ShippingLabel, at index: Int) -> AggregateOrderItem? {
        guard let orderItems = orderItemsByShippingLabelID[shippingLabel.shippingLabelID] else {
            return nil
        }

        return orderItems[safe: index]
    }
```

```suggestion
        guard let orderItem = shippingLabelOrderItemsAggregator?.orderItems(of: shippingLabel at: indexPath.row) else {
```

Reply:
yea sounds nice to encapsulate `safe` in a function, updated in https://github.com/woocommerce/woocommerce-ios/pull/3197/commits/c85b4c7e7b360c08c6038bad1dadacf1201f8f03